sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/DoFnTesterTest.java

```diff
@@ -405,6 +425,18 @@ public void processElement(ProcessContext c) throws Exception {
     }
   }
```

Comment:
<!--new_thread; commit:2b7b6939227e3e105feee6eddf1d6c943c40815a; resolved:0-->
Add a failure test showing that you can't access the side input during startBundle/finishBundle.